features/sign_me_up_to_the_account.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Signing up to the account
+   as a User
+   In order to use the application
+   I would like to login
```

Comment:
Please use capital letters at the beginning of sentences. And also, the file name should IMO be written in the same form as the user story